go.mod

```diff
@@ -1,6 +1,11 @@
 module github.com/googleapis/gax-go
 
 require (
+	github.com/golang/protobuf v1.2.0
```

Comment:
v1.3.0 is out. Not sure if we want to look into using that? Maybe in a diff PR

Reply:
Tracked in https://github.com/googleapis/google-cloud-go/issues/1341